unlock-protocol-com/blog/gitcoin-round-15.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Gitcoin Grants Round 15
+subTitle: Unlock is funding public goods with UDT!
+authorName: Angela Steffens
+publishDate: October 5, 2022
+description: Unlock Labs 2nd Gitcoin round matched $25k in UDT tokens for projects in our ecosystem and another $10k for Diversity, Equity & Inclusion (DEI).
+image: /images/blog/gitcoin-round-15.png
+---
+
+We were happy to participate in 
+[Gitcoin's Grants Round 15](https://gitcoin.co/grants/) as a sponsor of a $25k 
+pool for the greater Unlock ecosystem and another $10k added to the larger DEI 
+pool! There are some really exciting numbers to share about what we saw this 
+round. Gitcoin distributed $4.2 million dollars in funding and saw contributions 
+from 39.6k community donors. Unlock Protocol doubled the number of grantees in 
+our ecosystem going from 14 to 28.
+
+![Gitcoin's GR 15](/images/blog/gitcoin-round-15.png)
+
+Not only were we excited to see the number of grant applications increase in our
+ecosystem we were really excited to see the diversity in the types of initiatives 
+tagging us in. 
+
+## Climate Change
+We're sure that launching Unlock Protocol on the [Celo](https://celo.org/) 
+network is why we saw many more projects this round focused on climate change. 
+[Bloom Network](https://bloomnetwork.org/) continues to be the most well 
+supported project in both climate change and in the entire ecosystem. So far 
+they've raised more than $81k through Gitcoin Grants and we're so glad to see the 
+progress they've made with that funding. Some other projects with great potential 
+for climate impact are [Green Academy](https://green-academy.webflow.io/), 
+[Plants & Pillas](https://www.plantsandpillars3.com/), 
+[dMeter](https://dmeter.super.site/) and [Foresta](https://foresta.live/).
+
+## DeSci
+
+Some impressive projects leveraging Unlock Protocol for decentrailized science, 
+or DeSci, tagged us in this round. 
+[Omni Analytics](https://twitter.com/omnianalytics) maintaining the top spot for 
+community support in this category, has continued to use the funding they've 
+recieved by adding more content to their "Learning Data Science on the Ethereum 
+Blockchain" course. Congratulations to 
+[Active Inference Institute](https://www.activeinference.org/) who caught 
+community attention with their plans to create a first of it's kind open source 
+package Active Blockference which aims to implement Active Inference simulations 
+for congnitive modeling for the blockchain space, use Active Blockreference to 
+understand behavior in ETH/Web3 ecosystems and improve educational materials 
+around use of Active Blockreference. 
+
```

Comment:
line 45 has Blockference and lines 46 and 48 have Blockreference  — are those all correct? (it's spelled differently on line 45 vs the others)